platform/broadcom/docker-syncd-brcm/syncd.sh

```diff
@@ -1,12 +1,122 @@
 #!/usr/bin/env bash
+#
+# Based on /etc/init.d/syncd
 
-function clean_up {
-    service syncd stop
-    exit
+CMD_SYNCD=/usr/bin/syncd
+
+CMD_DSSERVE=/usr/bin/dsserve
+CMD_DSSERVE_ARGS="$CMD_SYNCD --diag"
+
+ENABLE_SAITHRIFT=0
+
+PLATFORM_DIR=/usr/share/sonic/platform
+HWSKU_DIR=/usr/share/sonic/hwsku
+
+if [ -x $CMD_DSSERVE ]; then
+    CMD=$CMD_DSSERVE
+    CMD_ARGS=$CMD_DSSERVE_ARGS
+else
+    CMD=$CMD_SYNCD
+fi
+
+parse_yaml() {
```

Comment:
It would be better, however, the syncd dockers are currently built from docker-base, not docker-config-engine, so they don't have sonic-cfggen available. I plan to create a new PR that will do this.